src/mixin-tester.vue

```diff
@@ -0,0 +1,26 @@
+<template>
+  <div class="d-none" />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import {
+  AllowableActionsMixin, AuthApiMixin, DateMixin, EnumMixin, LegalApiMixin, PayApiMixin
+} from '@/mixins'
+
+@Component({
+  mixins: [
+    AllowableActionsMixin,
+    AuthApiMixin,
+    DateMixin,
+    EnumMixin,
+    LegalApiMixin,
+    PayApiMixin
+  ]
+})
+export default class MixinTester extends Vue {}
+</script>
+
+<style>
+</style>
```

Comment:
This file was moved from the unit test folder to here so it could see the "shim" files and be parsed correctly by VS Code / Volar.

(The shim files let the Vue Language Features plugin know how to interpret some code.)